src/helpers/match-stack.js

```diff
@@ -7,7 +7,18 @@ import { toRegExp } from './string-utils';
  * @param {string} stackTrace - script error stack trace
  * @returns {boolean}
  */
+// https://github.com/AdguardTeam/Scriptlets/issues/226
+// eslint-disable-next-line import/no-mutable-exports, func-names
+export let shouldAbortStack = function () { };
```

Comment:
why as a function? 

just `export let shouldAbortStack = false;` seems to work fine. and no eslint exception needed above

Reply:
and move it please before jsdoc comment related to `matchStackTrace`

Reply:
> why as a function?
> 
> just `export let shouldAbortStack = false;` seems to work fine. and no eslint exception needed above

I wanted to use it as a variable, but when it's not used as a function, then it's removed from script when script is compiled/builded, and then it's exposed in a global scope. So that's why I used this as a function.
Similar issue occurs with something like this:
```
export let someObject = {};
```
but in this case, build fails.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/180991147-52159796-9ca4-4095-b7fe-ef0c5060fea0.png)

</details>

Maybe I'm doing something wrong, so I would appreciate any tips/helps with this.

Reply:
you're right, it should be a function type